src/views/v2/month/event-multiday.php

```diff
@@ -15,25 +15,55 @@
 
 $event    = $this->get( 'event' );
 $event_id = $event->ID;
+$day_number = $this->get( 'day' );
 
 $classes = [ 'tribe-events-calendar-month__event-multiday' ];
+$classes_inner = [ 'tribe-events-calendar-month__event-multiday-inner' ];
 
-if ( $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+// Check if it's featured.
+if ( $is_featured = isset( $event->featured ) && $event->featured ) { // @todo: later use tribe( 'tec.featured_events' )->is_featured( $event_id ) or similar
 	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
 }
+
+// If it starts today and this week, let's add the left border and set the width
+if ( $starts_today = $event->start_date == $day_number ) { // @todo:later we can check mm/dd or even year
+
+	// @todo: check if it ends this week or not, in order to split the duration
+	$classes[] = 'tribe-events-calendar-month__event-multiday-width-' . $event->duration;
+
+	if ( isset( $event->start_this_week ) && $event->start_this_week ) {
+		// adding the left border because it starts today
+		$classes_inner[] = 'tribe-events-calendar-month__event-multiday-inner--border-left';
+		$classes[] = 'tribe-events-calendar-month__event-multiday--start';
+	}
+
+} else {
+	$classes[] = 'tribe-events-calendar-month__event-multiday--hidden';
+}
+
+// if it ends this week, let's add the right border
+$end_this_week = isset( $event->end_this_week ) && $event->end_this_week;
+if ( $end_this_week ) {
+	$classes_inner[] = 'tribe-events-calendar-month__event-multiday-inner--border-right';
+	$classes[] = 'tribe-events-calendar-month__event-multiday--end';
```

Comment:
We should discuss this, as this was done elsewhere, but if we're adding a `--end` modifier, we shouldn't need the `--border-right` modifier on `tribe-events-calendar-month__event-multiday-inner` and just work off the `--end` modifier. Yes, we may be creating more another class to override this way, but the benefits are twofold: semantics and maintenance.

Reply:
`--end` is for the wrapper, `--border-right` is for the inner.

The wrapper needs to be curved in order to match the design, except if it's an event that continues next week. In that case the multiday edge on the right side should be just straight, and for that we need to modify the wrapper.

Reply:
Right, I mean to do something like this:

```
.tribe-events-calendar-month__event-multiday--end {

    .tribe-events-calendar-month__event-multiday-inner {
        // styles for --border-right here
    }
}
```

Reply:
I thought we were trying to avoid nesting. Are we making an exception on this one?

Reply:
Yes you're right, we are trying to avoid nesting. But in cases like this, the semantics surrounding this is specifically around the multiday event. We are saying that this is a case where we have an `--end` modifier to the multiday, that the multiday event component has an end cap. So given this case, we need to add all the styles that make sense for an `--end` modifier.

The difference between the above and having both `.tribe-events-calendar-month__event-multiday--end` and `tribe-events-calendar-month__event-multiday-inner--border-right` styles is around semantics and maintenance. For maintenance, now we have 2 places to make changes if we decide to change anything. For semantics, we could technically now have `tribe-events-calendar-month__event-multiday-inner--border-right` styles without `.tribe-events-calendar-month__event-multiday--end` styles, as these are 2 different modifiers. However, in the first example above, we can't have `tribe-events-calendar-month__event-multiday-inner--border-right` styles without `.tribe-events-calendar-month__event-multiday--end` as it is all controlled by 1 class, the `--end` modifier.

Hopefully these examples make sense. If not, let me know and we can hop on a call for it.

Reply:
Yes, I get it - But at some point I feel we're having cases where we can do the same for `--featured` for example, but still, and we're not doing it. Trying to find some clarity on where is that we should be doing the exceptions.

Reply:
We should be doing it for the `--featured` cases too. There were a couple cases earlier where it was done like this, but we should be sticking to only one class that affects the modified case.

Reply:
Cool, as long as we follow a pattern, I'm in, will change that.